examples/radio_settings.py

```diff
@@ -5,9 +5,33 @@
 if __name__ == '__main__':
 
     # Connect to radio
-    doodle1 = Doodle(ip="10.223.108.26", user="root", password="DoodleSmartRadio")
+    doodle1 = Doodle(ip="10.223.97.204", user="user", password="DoodleSmartRadio")
+
     doodle1_connected = doodle1.connect()
-    print(doodle1_connected) 
+    
+    doodle1.get_availible_channel_widths()
+    doodle1.get_availible_submodels()
+
+    # # Get the rssi values
+    doodle1.get_channel_info()
+    doodle1.get_frequency()
+
+    print("Frequency:", doodle1._frequency)
+    print("Channel:", doodle1._channel)
+    print("Channel Width:", doodle1._channel_width)
+    print("Version:", doodle1.get_firmware_version())
+
+    print('------------------------')
+
+    # Set the frequency, channel, bandwidth
+    doodle1.set_frequency(915)
```

Comment:
this example fails when the radio is not already on 915mhz.

we might need to 

1. find a way to get the correct channel for the specified frequency
2. OR change the frequency, get the new channels, and then update the channel and bandwidth from there